src/tv2-common/getSegment.ts

```diff
@@ -327,6 +327,8 @@ export async function getSegmentBase<ShowStyleConfig extends TV2ShowStyleConfig>
 			))
 	) {
 		blueprintParts[0].part.budgetDuration = totalTimeMs
+		/*NOTE: Set budgetDuration to Segment to new sofie-server*/
+		;(segment as any).budgetDuration = totalTimeMs
```

Comment:
@KvelaGorrrrnio Do we want to have this typed?
We are moving away from these Blueprints _relatively_ soon...

Reply:
I was the one suggesting to simply use any here, since it is a temporary fix for it.

Reply:
I agree